data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/IdsMultipartSender.java

```diff
@@ -61,12 +61,13 @@ public class IdsMultipartSender {
     private IdentityService identityService;
     private ObjectMapper objectMapper;
 
-    public IdsMultipartSender(@NotNull Monitor monitor, @NotNull OkHttpClient httpClient, @NotNull IdentityService identityService,
+    public IdsMultipartSender(@NotNull Monitor monitor, @NotNull OkHttpClient httpClient,
+                              @NotNull IdentityService identityService,
                               @NotNull ObjectMapper objectMapper) {
-        this.monitor = monitor;
-        this.httpClient = httpClient;
-        this.identityService = identityService;
-        this.objectMapper = objectMapper;
+        this.monitor = Objects.requireNonNull(monitor);
```

Comment:
I lost this modification, I would remove this checks and the `@NotNull` annotation as they provide no value, the not-null constraint is enforced by the injection model at the extension level

Reply:
removed them